Assets/Modules/Hero/Scripts/Hero.cs

```diff
@@ -0,0 +1,36 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using Aloha.EntityStats;
+
+namespace Aloha.Hero
+{
+    public class Hero<T> : Entity<HeroStats> where T : HeroStats
+    {
+        public float defense;
```

Comment:
Pareil ici, pourquoi répéter la variable defense alors qu'on peut utiliser this.stats.defense